packages/@lwc/engine-core/src/framework/modules/attrs.ts

```diff
@@ -15,29 +15,32 @@ import {
 import { RendererAPI } from '../renderer';
 
 import { EmptyObject } from '../utils';
-import { VBaseElement, VStatic } from '../vnodes';
+import { VBaseElement, VStatic, VStaticPart } from '../vnodes';
 
 const ColonCharCode = 58;
 
 export function patchAttributes(
-    oldVnode: VBaseElement | null,
-    vnode: VBaseElement,
+    oldnode: VBaseElement | VStaticPart | null,
+    node: VBaseElement | VStaticPart,
```

Comment:
In the browser, attribute patching for static parts is the same as with other elements.